downstream/modules/devtools/proc-self-service-initial-rbac-setup.adoc

```diff
@@ -7,8 +7,8 @@
 After you install {SelfServiceShort} and synchronize it with {PlatformNameShort},
 only users with admin privileges can view the auto-generated templates.
 
-You must configure initial Role-Based Access Control (RBAC) in {SelfServiceShort}
-to enable users who do not have admin privileges to view and execute auto-generated templates.
+You must configure initial Role-Based Access Control (RBAC) in the {SelfServiceShort}. 
+This configuration is mandatory for all users, without it, the {PlatformNameShortS} admin cannot view templates.
```

Comment:
Just a minor nit on punctuation (see [Semicolons between independent clauses](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-semicolons)) : 
```suggestion
This configuration is mandatory for all users; without it, the {PlatformNameShortS} admin cannot view templates.
```
Feel free to ignore if you're inclined to.